beacon-chain/rpc/eth/validator/handlers_test.go

```diff
@@ -1373,6 +1374,119 @@ func TestGetAttestationData(t *testing.T) {
 	})
 }
 
+func TestProduceSyncCommitteeContribution(t *testing.T) {
+	root := bytesutil.PadTo([]byte("0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2"), 32)
+	sig := bls.NewAggregateSignature().Marshal()
+	messsage := &ethpbalpha.SyncCommitteeMessage{
+		Slot:           1,
+		BlockRoot:      root,
+		ValidatorIndex: 0,
+		Signature:      sig,
+	}
+	syncCommitteePool := synccommittee.NewStore()
+	require.NoError(t, syncCommitteePool.SaveSyncCommitteeMessage(messsage))
+	server := Server{
+		CoreService: &core.Service{
+			HeadFetcher: &mockChain.ChainService{
+				SyncCommitteeIndices: []primitives.CommitteeIndex{0},
+			},
+		},
+		SyncCommitteePool: syncCommitteePool,
+	}
+	t.Run("ok", func(t *testing.T) {
+		url := "http://example.com?slot=1&subcommittee_index=1&beacon_block_root=0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2"
+		request := httptest.NewRequest(http.MethodGet, url, nil)
+		writer := httptest.NewRecorder()
+		writer.Body = &bytes.Buffer{}
+
+		server.ProduceSyncCommitteeContribution(writer, request)
+		assert.Equal(t, http.StatusOK, writer.Code)
+		resp := &ProduceSyncCommitteeContributionResponse{}
+		require.NoError(t, json.Unmarshal(writer.Body.Bytes(), resp))
+		require.NotNil(t, resp)
+		require.NotNil(t, resp.Data)
```

Comment:
we should assert more info about the response. Can we check it matches what we want? maybe something from the message?